scripts/controller/projectsController.js

```diff
@@ -8,5 +8,23 @@
     $('#projects').fadeIn(500);
   };
 
+  projectsController.index = function(ctx, next) {
+    if(ctx.articles.length) {
+      projectView.renderProjects();
+    } else {
+      page('/');
+    }
+  }
+
+  projectsController.showByType = function(ctx, next) {
```

Comment:
This is the first method called when your select box experiences a change event.  What I think you were looking to do with this method is to find the projects associated with the change (maybe by doing some kind of filter on your projects array and then attaching them to the context object).